packages/lwc-language-server/src/javascript/__tests__/compiler.test.ts

```diff
@@ -211,72 +103,98 @@ it('linter returns empty diagnostics on correct file', async () => {
     expect(diagnostics).toEqual([]);
 });
 
-it('transform returns javascript metadata', async () => {
+it('mapLwcMetadataToInternal returns expected javascript metadata', async () => {
```

Comment:
We significantly augmented the fixture used in this test. Every variety of decorator that we could think of was added, which is reflected in the changes below.